lib/cmd/fh3/app/clean-clone.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+var millicore = require("../../../utils/millicore.js");
+var fhreq = require("../../../utils/request");
+
+module.exports = cleanClone;
+
+cleanClone.usage = "fhc app clean-clone <appId>";
+cleanClone.desc = i18n._("Performs a clean clone of a git repo within the studio to resolve potential git issuse");
+
+function cleanClone (argv, cb) {
+  var appId = argv.appId;
+  var payload = {
+    clean: true
+  };
+
+  if (!appId) {
+    return cb(
+      new Error(i18n._('appId is required to perform a clean clone'))
+    );
+  }
+
+  millicore.widgForAppId(appId, function (err, widgId) {
+    if (err) {
+      return cb(err);
+    }
+    //Sample URL: https://domain.feedhenry.com/box/api/connectors/{widget}/apps/{appId}/pull
+    fhreq.POST(fhreq.getFeedHenryUrl(), "/box/api/connectors/" + widgId + '/apps/' + appId + "/pull", payload, function (err, data) {
```

Comment:
Switch to use `/box/api/projects/` instead, `/box/api/connectors/` is for services, although I believe both will work.